app/views/devise/invitations/new.html.erb

```diff
@@ -38,7 +38,7 @@
       <% end %>
 
       <% if (params[:role] == SiteCoordinatorRole::TYPE) || (params[:role] == TeamMemberRole::TYPE) %>
-        <div class="form-group">
+        <div class="form-group tagify-clickable-dropdown-wrapper">
```

Comment:
had to add this b/c the drop down arrow is inserted through css via an ::after classification, this will wrap the whole select box to ensure we include it

Reply:
is the drop down arrow inserted somewhere outside our code via tagify?

Reply:
for the single select (used for coalition and org leads) Tagify inserts it but for this one, multiselect, its inserted via honey crisp @arinchoi03